libs/site-semantic-theme/src/semantic/thecoin/globals/site.variables

```diff
@@ -137,6 +137,7 @@
 @theCoinPaletteGray1     : #6F6571;
 @theCoinPaletteGray2     : #F2F2F2;
 @theCoinPaletteGray3     : #767676;
+@theCoinPaletteGray4     : #6A6A6A;
```

Comment:
A grey that must have fell through a crack.

Reply:
hahaha - we've ended up with quite a large pallet :-).

Gray1 is very similar to Gray4.  What would you think if we got rid of one of them?

Reply:
We will have to test that that could look very bad some places